### ISSUE TITLE
ANDI used to work in our old build but in our latest one, we're getting error like "Refused to load script bec it violates Content Security Policy directive"

### ISSUE THREAD

**Issue by knottydon** [SSAgov/ANDI#197]
We use ANDI for accessibility of our product but in our current build, ANDI doesn't work anymore.  It used to run in the last version.  I checked the Console and I'm getting the error attached in this issue:

![2023-05-05 09_37_41-Inbox - Don Balan@teloworksinc com - Outlook](https://user-images.githubusercontent.com/2860498/236362045-507f6100-d95d-45fa-900f-3303a23884c3.png)

Then we tried to run again ANDI in the build that was working before but now, it also doesn't run in that version.  What has changed in ANDI? What can we do to fix this problem? Thank you for your replies.

**Comment by JohnCotterSSA** [comment]
Hi. Nothing changed in ANDI that would cause that console message. What changed is that the website you are testing has added a CSP directive which prevents external scripts (like ANDI) from executing.

Here is info from the ANDI FAQ on what to do next: [ANDI Isn't Workin](https://www.ssa.gov/accessibility/andi/help/faq.html#wontLaunch)
